public/app/features/library-panels/state/api.ts

```diff
@@ -64,7 +64,7 @@ export async function getLibraryPanel(uid: string, isHandled = false): Promise<L
     schemaVersion: 35, // should be saved in the library panel
     panels: [result.model],
   });
-  const model = dash.panels[0].getSaveModel(); // migrated panel
+  const { scopedVars, ...model } = dash.panels[0].getSaveModel(); // migrated panel
```

Comment:
getSaveModel should never include scopedVars, it should be safe to add scopedVars to non persisted props in PanelModel? 

It is cleaned as part of DashboardModel.getPanelSaveModels but maybe it's better to do that in PanelModel ?